tests/test_xsd.py

```diff
@@ -21,24 +21,32 @@
 # ======================================================================================================================
 # Tests
 # ======================================================================================================================
-def test_happy_path_asc(testdir, properly_decorated_test_function):
+def test_happy_path_asc(testdir, properly_decorated_test_function, mocker):
     """Verify that 'get_xsd' returns an XSD stream that can be used to validate JUnitXML."""
 
+    # Mock
+    zz = mocker.MagicMock()
+    xmlpf = XmlParsingFacade(zz)
+
     # Setup
     testdir.makepyfile(properly_decorated_test_function.format(test_name='test_happy_path',
                                                                test_id='123e4567-e89b-12d3-a456-426655440000',
                                                                jira_id='ASC-123'))
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd()))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd()))
```

Comment:
Since an external program is accessing this method it should be marked as public.

Reply:
Can we do this later as its in a different repository